test/mongodb.test.js

```diff
@@ -1438,6 +1458,129 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('find should be able to query by location via near with maxDistance', function(done) {
+      var coords = { lat: 30.274085, lng: 120.15507000000002 };
+
+      geoDb.autoupdate(function(err) {
+        async.parallel([
+          createPosts(false),
+          createPosts(false),
+          createPosts(false),
+          createPosts(true),
+        ], function(err) {
+          if (err) return done(err);
+
+          PostWithLocation.find({
+            where: {
+              location: {
+                near: new GeoPoint(coords),
+                maxDistance: 17000,
+                unit: 'meters',
+              },
+            },
+          }, function(err, results) {
+            if (err) return done(err);
+            results.length.should.be.equal(3);
+
+            var dist = 0;
+            results.forEach(function(result) {
+              var currentDist = testUtils.getDistanceBetweenPoints(coords, result.location);
+              currentDist.should.be.aboveOrEqual(dist);
+              currentDist.should.be.belowOrEqual(17);
+              dist = currentDist;
+            });
+
+            done();
+          });
+        });
+      });
+    });
+
+    it('find should be able to query by location via near with minDistance set', function(done) {
+      var coords = { lat: 30.274085, lng: 120.15507000000002 };
+
+      geoDb.autoupdate(function(err) {
+        async.parallel([
+          createPosts(false),
+          createPosts(false),
+          createPosts(false),
+          createPosts(true),
+        ], function(err) {
+          if (err) return done(err);
+
+          PostWithLocation.find({
+            where: {
+              location: {
+                near: new GeoPoint(coords),
+                minDistance: 17000,
+                unit: 'meters',
+              },
+            },
+          }, function(err, results) {
+            if (err) return done(err);
+            should.exist(results);
```

Comment:
Actually you should leave this and verify the results are as expected. Meaning did you get the right values and number of objects back.